docs/labels.md

```diff
@@ -0,0 +1,22 @@
+# Project-specific labels
+
+## Component
+
+We naturally have a set of contracts here, so each contract has a label of its
+own (the same color is used for each except `neofs` since it's not an FS chain
+contract, they rarely mix in a single issue):
+
+- alphabet
+- audit
+- balance
+- container
+- neofs
+- neofsid
+- netmap
+- nns
+- proxy
+- reputation
+- subnet
+
+  Should not be used for new issues since the contract is gone, but kept for
```

Comment:
what this line is about?

Reply:
subnet, see https://github.com/nspcc-dev/neofs-contract/blob/c8c32311371cb592aa9bd2a1045b0ce14dd3973f/docs/labels.md

Reply:
well, no the best readability but ok